lib/phoneNumber.js

```diff
@@ -0,0 +1,61 @@
+var PhoneNumber = function (client) {
+	/**
+		 * Gets a list of all your phone numbers.
+		 * @param {Object} params Query parameters for sorting the results
+		 * @param {String} [params.page=0] Used for pagination to indicate the page
+	   *  requested for querying a list of phone numbers. If no value is specified the default is 0.
+		 * @param {String} [params.size=25] Used for pagination to indicate the size
+	   *  of each page requested for querying a list of phone numbers.
+	   *  If no value is specified the default value is 25. (Maximum value 1000)
+		 * @param {String} [params.applicationId] Used to filter the retrieved list
+	   *  of numbers by an associated application ID.
+		 * @param {String} [params.state] Used to filter the retrieved list of numbers
+	   *   allocated for the authenticated user by a US state.
+		 * @param {Number} [params.name] Used to filter the retrieved list of numbers
+	   *  allocated for the authenticated user by it's name.
+		 * @param {Number} [params.city] Used to filter the retrieved list of numbers
+	   *  allocated for the authenticated user by it's city.
+	   * @param {Number} [params.numberState] Used to filter the retrieved
+	   *  list of numbers allocated for the authenticated user by the number state.
+		 * @param {Function} callback A callback with the list of numbers
+		 * @return {Promise} A promise for the list of numbers
+		 */
+	this.list = function (params, callback) {
```

Comment:
Why is `list` an instance of PhoneNumber? I would have to create a PhoneNumber object just to just my PhoneNumbers?